src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -126,6 +126,10 @@ def _polish_bad_errors(ex):
 cli_command(__name__, 'webapp deployment user set', custom_path + 'set_deployment_user')
 cli_command(__name__, 'webapp deployment list-publishing-profiles',
             custom_path + 'list_publish_profiles')
+cli_command(__name__, 'webapp deployment container config',
+            custom_path + 'enable_cd')
+cli_command(__name__, 'webapp deployment container show-container-cd-url',
```

Comment:
Suggest `show-cd-url`. `container` is redundant under a category with the same name